src/test/java/web/CartTest.java

```diff
@@ -0,0 +1,55 @@
+package web;
+
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+import com.zebrunner.carina.core.registrar.ownership.MethodOwner;
+import com.zebrunner.carina.core.registrar.tag.Priority;
+import com.zebrunner.carina.core.registrar.tag.TestPriority;
+import java.util.List;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import web.components.CartProduct;
+import web.components.NavBar;
+import web.components.ProductCard;
+
+public class CartTest extends AbstractWebTest {
+
+    @DataProvider(name = "ProductsList")
+    public Object[][] dataProviderProducts() {
+        return new Object[][]{
+            {"Samsung galaxy s6"},
+            {"Nokia lumia 1520"},
+            {"Sony vaio i5"},
+        };
+    }
+
+    @Test(dataProvider = "ProductsList")
+    @TestPriority(Priority.P1)
+    @MethodOwner(owner = "jinocencio-solvd")
+    public void testAddToCart(String productTitleToAdd) {
+        // Add product to cart
+        ProductCard productCard = homePage.getProductCardByTitle(productTitleToAdd);
+        ProductPage productPage = productCard.clickProductTitle();
+        productPage.assertPageOpened();
```

Comment:
please change to assertTrue(isPageOpened(), message)
```suggestion
        productPage.assertPageOpened();
```